src/bin/server.ts

```diff
@@ -10,10 +10,21 @@ interface Args {
   _: Array<string>;
   config?: string;
   "mock-driver": boolean;
+  "websocket-port": number;
 }
 
 (async () => {
-  const args = parseArgs<Args>(["_", "config", "mock-driver"]);
+  const args = parseArgs<Args>([
+    "_",
+    "config",
+    "mock-driver",
+    "websocket-port",
```

Comment:
We only use a single port, so I think that just `port` suffices

Reply:
Will update this too